src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,45 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+
+import static ladder.domain.LadderGame.MINIMUM_PARTICIPANTS;
+
+public class Line {
+    private List<Boolean> points = new ArrayList<>();
+    private Random random = new Random(System.nanoTime());
+
+    private Line(int countOfPerson) {
+        random.setSeed(System.nanoTime());
+        validateCountOfPerson(countOfPerson);
+        points.add(random.nextBoolean());
+        while (points.size() != countOfPerson - 1) {
+            addRandomValue();
```

Comment:
(!) 스타일 일수도있습니다.. 😄 
객체에 무슨 행동을 가하는지, 좀더 명확하게 드러내는것도 고려해보면 어떨까요?

\# 왜컬렉션 유틸에 마지막꺼 꺼내는건 없는지 .. 항상 아쉬운것같습니당.. ㅠㅠ

```suggestion
            poinsts.add(getRandomValue(poinst.get(points.size() -1));
```

```suggestion
           addRandomValue(this.points)
```

Reply:
이 부분은 조금 더 고려해보겠습니다. 확실히 addRandomValue가 실제로 어떤값에 영향을 끼치는 지 나타나지 않아서 명확성이 부족해보이긴하네요